### PR TITLE
added frontend test cases in typescript

### PR DESCRIPTION
**Pull request by Shrasti0207**

### CODE REVIEW

**Review by Ritika-gupta20 — APPROVED** [review]

**Review by Sparsh79 — APPROVED** [review]

File: go1percentTS/src/test/Go1PercentFEAutomation/techhub/EmployeeUserTest1.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    'Verify user should be able to see blank fields error messages': (browser: NightwatchBrowser) => {
+        var blankFields = browser.page.techhub.EmployeeUser.techHubPage();
+        blankFields
+            .draftAndPendingButton()
+            .requiredTitle()
+            .assert.textContains('@errorMessage', 'Title is Required')
+            .requiredCaption()
+            .assert.textContains('@captionRequiredErrorMessage', 'Caption is Required')
+            .execute(function() {
+                document.querySelector('textarea[formcontrolname="description"]')?.scrollIntoView();
```
Comment (deepansh257) [inline go1percentTS/src/test/Go1PercentFEAutomation/techhub/EmployeeUserTest1.ts:129]: Add locator in page object and call from there. Do not add locators in test file.

File: go1percentTS/src/test/Go1PercentFEAutomation/techhub/EmployeeUserTest1.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+            .end()
+    },
+
+    //error messages for required category and language
+    'Verify user should be able to see Category and Language is required error message':(browser: NightwatchBrowser) => {
+        var categoryAndLanguageError = browser.page.techhub.EmployeeUser.techHubPage();
+        categoryAndLanguageError
+        .draftAndPendingButton()
+        .execute(function () {
+            document.querySelector('select[id="category"]')?.scrollIntoView();
```
Comment (deepansh257) [inline go1percentTS/src/test/Go1PercentFEAutomation/techhub/EmployeeUserTest1.ts:150]: Create a reusable method to scroll to a particular element and use that everywhere needed

**Review by deepansh257 — CHANGES_REQUESTED** [review]

File: go1percentTS/src/test/Go1PercentFEAutomation/techhub/EmployeeUserTest1.ts (hunk 1 above)
Comment (Shrasti0207) [reply go1percentTS/src/test/Go1PercentFEAutomation/techhub/EmployeeUserTest1.ts:129]: querySelector is a core javascript method. Hence it is not picking the locators from POM

File: go1percentTS/src/test/Go1PercentFEAutomation/techhub/EmployeeUserTest1.ts (hunk 2 above)
Comment (Shrasti0207) [reply go1percentTS/src/test/Go1PercentFEAutomation/techhub/EmployeeUserTest1.ts:150]: we can not create a reusable method in POM of execute function

**Review by deepansh257 — APPROVED** [review]

**Pull request merged**